### DISCUSSION TITLE
Airnav xrange

### DISCUSSION THREAD

**Discussion by abpei** [vradarserver/vrs#114] [General]
Hello,

How is the xrange url supposed to be entered for preview 11? If I point the url directly to the json, I get bad messages. Anything else is no messages. The format still looks to be the same as in the [old form posts](https://forum.virtualradarserver.co.uk/viewtopic.php?t=1556).

Wizard
![image](https://github.com/vradarserver/vrs/assets/5302376/48ea93f4-838b-40be-b7d4-4e23061edd96)

Result
![image](https://github.com/vradarserver/vrs/assets/5302376/a6f5cb2b-9502-46b3-948b-98030e5d7442)

Example dump attached.
[aircraft.json](https://github.com/vradarserver/vrs/files/12904564/aircraft.json)

Thanks,
~A

**Comment by vradarserver** [comment]
The feed looks OK. If you double-click the receiver it should give you a statistics dialog, and that should hopefully give you some clue as to what it's complaining about.

**Comment by abpei** [comment]
Thanks for your reply, appreciated.
I'm getting all packages 'badly formatted'.
Any suggestions?
